core/src/main/java/org/javarosa/core/api/ClassNameHasher.java

```diff
@@ -0,0 +1,25 @@
+package org.javarosa.core.api;
+
+import org.javarosa.core.util.externalizable.Hasher;
+
+
+/**
+ * Most simple possible "hasher" - just gets the byte representation of the hash name
+ *
+ * @author wspride
+ */
+public class ClassNameHasher extends Hasher {
+
+    private final static int CLASS_HASH_SIZE = 32;
```

Comment:
Question: Is this # valid anymore? The hash of the name is basically as long as it wants to be, right? Is it worth considering truncating?